src/e2e-test/features/BQExecuter/BQExecuteConnectorErrorScenarios.feature

```diff
@@ -0,0 +1,47 @@
+@BQExecute
+Feature: Verify BigQuery Execute Query error scenarios
+
+  @BQExecute
+  Scenario Outline: Verify BigQuery Execute Query error for mandatory fields with blank SQL Query
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "BigQuery Execute" from the plugins list as: "Action"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Validate mandatory property error for "<property>"
+    Examples:
+      | property |
+      | sql      |
+
+  @BQExecute
+  Scenario: Verify BigQuery Execute Query with Blank Dataset and Table name
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "BigQuery Execute" from the plugins list as: "Action"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Enter the BigQuery Execute query "bqexecuteQueryUC1"
```

Comment:
BigQuery Execute -> BigQueryExecute

Change in all steps

Reply:
@sawantpritam -Done